packages/azure-http-specs/specs/azure/resource-manager/resources/location.tsp

```diff
@@ -0,0 +1,175 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@typespec/spector";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using Azure.Core;
+using Azure.ResourceManager;
+using TypeSpec.OpenAPI;
+using Spector;
+
+namespace Azure.ResourceManager.Resources;
+
+@resource("locationResources")
+@parentResource(ResourceGroupLocationResource)
+model LocationResource is ProxyResource<LocationResourceProperties> {
+  ...ResourceNameParameter<LocationResource>;
+}
```

Comment:
Any reason we uses `ResourceGroupLocationResource`?

So far, the only real use of it is `ArmLocationResource<"Subscription">`, which is `SubscriptionLocationResource`
```
@parentResource(ArmLocationResource<"Subscription">)
model QuotaResource is ProxyResource<QuotaResourceProperties> {
```

Reply:
Fixed by the [commit#9562e90](https://github.com/Azure/typespec-azure/pull/1932/commits/9562e903d2dbcdf451d595dba901060122cc15a1) and [commit#587687e](https://github.com/Azure/typespec-azure/pull/1932/commits/587687e1d0ee677125f86c6346fc1e417413d79e)